GitCommands/Git/GitVersion.cs

```diff
@@ -22,7 +22,7 @@ public class GitVersion : IComparable<GitVersion>
         private static readonly GitVersion v2_19_0 = new GitVersion("2.19.0");
         private static readonly GitVersion v2_20_0 = new GitVersion("2.20.0");
 
-        public static readonly GitVersion LastSupportedVersion = v2_11_0;
+        public static readonly GitVersion LastSupportedVersion = v2_19_0;
         public static readonly GitVersion LastRecommendedVersion = new GitVersion("2.25.1");
```

Comment:
Should be bumped to 2.27 - what we should use when "testing"

Reply:
Sure